packages/components/package.json

```diff
@@ -33,7 +33,8 @@
     "@emotion/styled": "^11",
     "classnames": "^2.2.6",
     "cross-fetch": "^3.1.4",
-    "framer-motion": "^4"
+    "framer-motion": "^4",
+    "web3": "^1.6.1"
```

Comment:
We already have `ethers`, can you use that instead?

Reply:
cool, am checking 🙂